src/components/manage/Toolbar/More.jsx

```diff
@@ -126,6 +146,58 @@ class More extends Component {
     document.removeEventListener('mousedown', this.handleClickOutside, false);
   };
 
+  componentDidUpdate(prevProps, prevState) {
+    let erroredAction = '';
+    if (
+      prevProps.workingCopy.apply.loading &&
+      this.props.workingCopy.apply.error
+    ) {
+      erroredAction = 'apply';
+    } else if (
+      prevProps.workingCopy.create.loading &&
+      this.props.workingCopy.create.error
+    ) {
+      erroredAction = 'create';
+    } else if (
+      prevProps.workingCopy.remove.loading &&
+      this.props.workingCopy.remove.error
+    ) {
+      erroredAction = 'remove';
+    }
+
+    if (erroredAction) {
+      if (this.props.workingCopy[erroredAction].error.status === 401) {
```

Comment:
The initial state from redux is
```
const initialState = {
  info: {},
  apply: {
    loaded: false,
    loading: false,
    error: null,
  },
  create: {
    loaded: false,
    loading: false,
    error: null,
  },
  remove: {
    loaded: false,
    loading: false,
    error: null,
  },
};
```
So I think this code is always safe, since `erroredAction` is there when `this.props.workingCopy[erroredAction].error` is truthy. The only thing that would break this would be changing the reducer to save something different than an object in that `error` in the state, but that wouldn't be caught by a `?`.